earth_enterprise/src/common/notify.cpp

```diff
@@ -68,6 +69,52 @@ khNotifyLevel getNotifyLevel() {
   return notifyLevel;
 }
 
+std::string khNotifyLevelToString(khNotifyLevel level)
+{
+    std::string retval;
+    switch(level)
+    {
+    case NFY_FATAL: retval = "NFY_FATAL"; break;
+    case NFY_WARN: retval = "NFY_WARN"; break;
+    case NFY_PROGRESS: retval = "NFY_PROGRESS"; break;
+    case NFY_INFO: retval = "NFY_INFO"; break;
+    case NFY_INFO2: retval = "NFY_INFO2"; break;
+    case NFY_DEBUG: retval = "NFY_DEBUG"; break;
+    case NFY_VERBOSE: retval = "NFY_VERBOSE"; break;
+    case NFY_NOTICE:
+    default: retval = "NFY_NOTICE";
+    };
+    return retval;
+}
+
+khNotifyLevel stringTokhNotifyLevel(std::string level)
```

Comment:
Consider using `const std::string &` for the parameter.

Reply:
Agreed. Changing.